jobs/models.py

```diff
@@ -8,36 +8,60 @@ class JobType(models.TextChoices):
     INTERNSHIP = '3', 'Internship'
 
 
+class Location(models.TextChoices):
+    Tel_Aviv = '1', 'Tel Aviv'
+    Jerusalem = '2', 'Jerusalem'
+    Haifa = '3', 'Haifa'
+
+
 # Jobs Model
 class Job(models.Model):
     title = models.CharField(max_length=100)
     description = models.CharField(max_length=255)
-    location = models.CharField(max_length=100)
+    location = models.CharField(
+        max_length=15,
+        choices=Location.choices,
+        default=Location.Tel_Aviv,
+        blank=True, null=True
+    )
     job_type = models.CharField(
         max_length=15,
         choices=JobType.choices,
         default=JobType.PART_TIME,
         blank=True, null=True
     )
     company_name = models.CharField(max_length=255)
-    company_description = models.CharField(max_length=255)
     post_until = models.DateField()
     is_active = models.BooleanField()
+    marked_count = models.IntegerField(default=0)
+    apply_link = models.URLField(unique=True, null=True)
 
     # Function to create a new job
     @classmethod
-    def create_job(cls, title, description, location, job_type, company_name,
-                   company_description, post_until, is_active):
+    def create_job(cls, title, description, location, job_type,
+                   company_name, post_until, is_active, marked_count, apply_link):
         job = cls(title=title, description=description, location=location,
                   job_type=job_type, company_name=company_name,
-                  company_description=company_description, post_until=post_until, is_active=is_active)
+                  post_until=post_until, is_active=is_active, marked_count=marked_count, apply_link=apply_link)
         job.save()
         return job
 
+    def del_job(self):
+        try:
+            self.job.delete()
+        except Job.DoesNotExist:
+            return False
+        return True
+
     # Function to reduce the length of description
     def snippet_description(self):
         return self.description[:25] + '...'
 
     # Jobs title to string - return the title of the job
     def __str__(self) -> str:
         return self.title
+
+    @staticmethod
+    # Function to search job by title
+    def get_jobs_by_title(title):
```

Comment:
> PR description:
> 
> This PR will add new attributes to the Job class such marked_count and apply_link to job. Also edit existing attributes such as location. In addition, it will include methods and tests for creating a new job and deleting existing job, and retrieve the title/description of the job.
> 
> The main functionality will be to get the basic job app data model implemented so we can later build our main features around it in the following PR. Such as Job search and maybe Save Job.
> 
> Each HR user(recriter) will be able to publish a job and delete a job. Each Student user will be able in a future PR to search for a job dependent on its location, title and job type. Also a Student user can click on the apply_link displayed in each job and will be redirected to the company website to apply for the selected job. The Student user will be able in the future to mark (Save) a job so it will be saved to their profile. The HR user who published the job will be able to see from the marked_count attribute how many Student users marked (Saved) their published job. @ormergi

FYI you can edit the description and add this